config/canonical.json

```diff
@@ -10964,7 +11018,7 @@
   },
   "amenity/fuel|Puma": {
     "count": 604,
-    "nomatch": ["shop/clothes|Puma"],
+    "nomatch": ["shop/clothes|Puma","shop/sports|Puma"],
```

Comment:
You can remove the nomatch for sports|Puma now

Reply:
Fixed